opencga-catalog/pom.xml

```diff
@@ -117,6 +117,7 @@
         <dependency>
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-solrj</artifactId>
+            <scope>test</scope>
```

Comment:
This and the other solr-related dependencies are not needed.
I can only see one usage at `org.opencb.opencga.catalog.managers.StudyManagerTest` , and it should be replaced with comons-lang3

```java
import org.apache.solr.common.StringUtils;
```

```java
import org.apache.commons.lang3.StringUtils;
```

Reply:
Agree. To be changed in the next release.